app/views/requirements/show.html.erb

```diff
@@ -14,7 +14,7 @@
   <strong>Contact Info:</strong>
   <%= @requirement.contact_name %> <br>
   <strong>Notes:</strong>
-  <%= @requirement.notes %>
+  <%= markdown(@requirement.notes) %>
```

Comment:
This is beyond the scope of this PR (it's not an e-mail). We don't currently include Markdown in our requirement notes (not that we couldn't), but we would also want to update the form to make it clear that admins can use Markdown formatting. Either way, this would be a separate issue / PR.